charts/harvester-cloud-provider/values.yaml

```diff
@@ -51,8 +49,13 @@ tolerations:
 - effect: NoExecute
```

Comment:
Few lines above, still needs nodeSelector to pick linux: https://github.com/rancher/charts/blob/dev-v2.6/charts/rancher-logging/rancher-logging/100.0.1%2Bup3.15.0/values.yaml#L41-L42